migrations/20201015005801-update-user-roles.js

```diff
@@ -1,57 +1,57 @@
 'use strict';
 
 module.exports = {
-  up: (queryInterface, Sequelize) => {
-    return queryInterface.sequelize.transaction((t) => {
-      return Promise.all([
-        queryInterface.addColumn(
-          'users',
-          'roleId',
-          { type: Sequelize.DataTypes.INTEGER },
-          { transaction: t }
-        ),
-        queryInterface.sequelize.query(
-          `UPDATE users SET "roleId" = 1 WHERE "accessLevel" = 'COMMANDER' OR "accessLevel" = 'ADMIN'`,
-          { transaction: t }
-        ),
-        queryInterface.sequelize.query(
-          `UPDATE users SET "roleId" = 2 WHERE "accessLevel" = 'SUPERVISOR'`,
-          { transaction: t }
-        ),
-        queryInterface.removeColumn('users', 'accessLevel', {
-          transaction: t,
-        }),
-      ]);
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (t) => {
+      await queryInterface.addColumn(
```

Comment:
Rather than having multiple awaits, shouldn't we be using `.then()` syntax? This guarantees that the initial promise finishes before moving on to the next one.

Reply:
i think you might be a bit confused - both `awaits` and `.then()` waits for the initial promise to resolve before proceeding

there is a case to be made to group some of the awaits to run synchronously if they don't need to be done sequentially, but the performance gain is relatively minor and it's not particularly important